deeplearning4j/deeplearning4j-nlp-parent/deeplearning4j-nlp/pom.xml

```diff
@@ -86,6 +86,12 @@
           <artifactId>commons-lang3</artifactId>
           <version>${commonslang.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.github.vinhkhuc</groupId>
+        <artifactId>jfasttext</artifactId>
+        <version>0.4</version>
+      </dependency>
```

Comment:
Also try to force a dependency on the same version of JavaCPP as used in the rest of Deeplearning4j.

Reply:
At this moment that's not our code, and as I see in maven central - 0.4 is the only available version at this moment.

Reply:
No, but newer versions of ND4J will not work with an old version of JavaCPP, while an old version of JFastText *might* work with a new version of JavaCPP.